bin/test-storybook.js

```diff
@@ -162,6 +171,10 @@ const main = async () => {
 
   process.env.TARGET_URL = targetURL;
 
+  if (runnerOptions.coverage) {
+    process.env.COLLECT_COVERAGE = 'true';
```

Comment:
I'm thinking maybe we should prefix these variables, e.g. `SB_COLLECT_COVERAGE`. WDYT?

Reply:
makes sense as discussed. Done!